Bonus/practicals/README.md

```diff
@@ -0,0 +1,9 @@
+# practicals
```

Comment:
Thank you for the effort.
This whole part would be more natural in a section: "StartingTeamProject" in the /Project directory.

Reply:
Should i move it there ?
What about creating a pull request in the /Project directory after fixing my typos ?

Reply:
Thank you for reviewing my commit. I fixed all the typos.